babel.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    presets: [
+      ['@babel/preset-env', {targets: {node: 'current'}}],
+      '@babel/preset-typescript',
+    ],
+};
```

Comment:
Using babel here to run jest ts tests, as they wouldn't run expect on ts@5, but ts@5 doesn't match with our deps